src/components/Card/Card.scss

```
@@ -71,6 +71,14 @@
   }
 }
 
+.Subsection {
+  + .Subsection {
+    margin-top: spacing();
+    padding-top: spacing();
+    border-top: border-width() solid color('sky');
```

is `border-width() solid color('sky')` effectively the same thing as `border()` ?

---

☝️ Yes, that's exactly what the default/base `border()` resolves to.
```suggestion
    border-top: border();
```

---

Agreed this should just be `border()`
```suggestion
    border-top: border();
```